base/linalg/qr.jl

```diff
@@ -26,7 +26,7 @@ end
 QRPivoted(factors::AbstractMatrix{T}, τ::Vector{T}, jpvt::Vector{BlasInt}) where {T} =
     QRPivoted{T,typeof(factors)}(factors, τ, jpvt)
 
-function qrfactUnblocked!{T}(A::AbstractMatrix{T})
+function qrfact!{T}(::Type{QR}, A::AbstractMatrix{T})
```

Comment:
perhaps can also update to the new where syntax 
`qrfact!(::Type{QR}, A::AbstractMatrix{T}) where T`